smpplib/client.py

```diff
@@ -65,15 +65,11 @@ class Client(object):
     sequence_generator = None
 
     def __init__(self, host, port, timeout=5, sequence_generator=None):
-        """Initialize"""
-
         self.host = host
         self.port = int(port)
         self._socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self._socket.settimeout(timeout)
-        if sequence_generator is None:
-            sequence_generator = SimpleSequenceGenerator()
-        self.sequence_generator = sequence_generator
+        self.sequence_generator = sequence_generator or SimpleSequenceGenerator()
```

Comment:
I think `is None` version is better here as people may want to override `__bool__` for things like sequence generators

Reply:
Yeah, makes sense. I'll change it back.